README.md

```diff
@@ -2,152 +2,108 @@
 
 This repository contains the official Account Abstraction - Hybrid Compute snap.
 
+[Hybrid Compute](https://docs.boba.network/hc) utilizes [Account Abstraction](https://docs.boba.network/developer/features/aa-basics) on Boba Network to engage with off-chain APIs seamlessly. This enables you to engage with GenAI or any other off-chain data you can imagine. Leverage Hybrid Compute to increase the power and interoperability of your MetaMask.
+
+Unlock the power of real-world data for Web3 with Hybrid Compute; a single line of code is all it takes.
+
 ## Audit
-The snap has been audited by Sayfer, [report](https://sayfer.io/audits/metamask-snap-audit-report-for-enya-labs/)
+The snap has been audited by Sayfer. You can find the report [here](https://sayfer.io/audits/metamask-snap-audit-report-for-enya-labs/).
+
+## Prerequisites
+
+Snaps is pre-release software. To interact with (your) Snaps, install:
+- [MetaMask Flask](https://metamask.io/flask/), a canary distribution for developers that provides access to upcoming features
+- `yarn`
+- `npm`
+
+## Cloning
+
+Clone the template-snap repository [using this template](https://github.com/MetaMask/template-snap-monorepo/generate) and set up your development environment.
```

Comment:
that's the tutorial for creating your own snap :-P. 
Goal is that developers understand how to use our snap.